tests/test_renderer.py

```diff
@@ -1,24 +0,0 @@
-import pytest
```

Comment:
Have these tests moved somewhere else?

Reply:
I only removed import statements when my linter told me that the import was unused.
If they were moved, I must assume they were moved prior to this PR

Reply:
It looks like the file `test_renderer.py` was removed (but is still present in `v0.1.1`)

Reply:
hm... https://github.com/leotrs/manim/commit/7d83b902c8dae03c66591a3ebeadc529c459305e
@leotrs ?

Reply:
That test was really only ever testing that the `video_quality_config` was behaving correctly, but this PR deleted video_quality_config so the test became obsolete.